systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicScalabilityUtils.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.kafkaUtils;
+
+import io.strimzi.api.kafka.model.KafkaTopicSpec;
+import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
+import io.strimzi.systemtest.templates.crd.KafkaTopicTemplates;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import io.strimzi.systemtest.resources.ResourceManager;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import io.strimzi.systemtest.enums.CustomResourceStatus;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+
+public class KafkaTopicScalabilityUtils {
+    /**
+     * This class contains crucial methods to create, modify and check large amount of kafka topics
+     * */
+
+    private static final Logger LOGGER = LogManager.getLogger(KafkaTopicUtils.class);
+    private KafkaTopicScalabilityUtils() {}
+
+    public static void createTopicsViaK8s(ExtensionContext extensionContext, String namespaceName, String clusterName, String topicPrefix,
+                                          int numberOfTopics, int numberOfPartitions, int numberOfReplicas, int minInSyncReplicas) {
+        LOGGER.info("Creating topics via Kubernetes");
+
+        for (int i = 0; i < numberOfTopics; i++) {
+            String currentTopicName = topicPrefix + i;
+            LOGGER.debug("Creating topic {}", currentTopicName);
```

Comment:
If we will create 1000 topics, in debug log will be 1000 lines of `Creating topic x`, which can overfill the `strimzi-debug.log` which can cause troubles. Wouldn't be better to log just bunch or none of the topics?